programs/compressed-pda/src/lib.rs

```diff
@@ -78,6 +78,11 @@ pub enum ErrorCode {
     LengthMismatch,
 }
 
+// // TODO(vadorovsky): Come up with some less glass chewy way of reusing
+// // our light-heap allocator if it's already used in some dependency.
+// #[cfg(all(feature = "custom-heap", target_os = "solana"))]
+// pub use account_compression::GLOBAL_ALLOCATOR;
```

Comment:
leftover?

Reply:
Ah yes, I actually found the less glass chewy way, lol. :joy: I'm just defining the custom allocator in the `heap` crate, I also moved all the logging logic there and to the macro.

Reply:
nice!